moped-database/metadata/tables.yaml

```diff
@@ -1,3 +1,172 @@
+- table:
+    name: moped_project_types
+    schema: deprecated
+  object_relationships:
+    - name: moped_type
+      using:
+        foreign_key_constraint_on: project_type_id
+  insert_permissions:
+    - role: moped-admin
+      permission:
+        check: {}
+        set:
+          created_by_user_id: x-hasura-user-db-id
+          updated_by_user_id: x-hasura-user-db-id
+        columns:
+          - is_deleted
+          - project_id
+          - project_type_id
+    - role: moped-editor
+      permission:
+        check: {}
+        set:
+          created_by_user_id: x-hasura-user-db-id
+          updated_by_user_id: x-hasura-user-db-id
+        columns:
+          - is_deleted
+          - project_id
+          - project_type_id
+  select_permissions:
+    - role: moped-admin
+      permission:
+        columns:
+          - created_at
+          - created_by_user_id
+          - id
+          - is_deleted
+          - project_id
+          - project_type_id
+          - updated_at
+          - updated_by_user_id
+        filter: {}
+    - role: moped-editor
+      permission:
+        columns:
+          - created_at
+          - created_by_user_id
+          - id
+          - is_deleted
+          - project_id
+          - project_type_id
+          - updated_at
+          - updated_by_user_id
+        filter: {}
+    - role: moped-viewer
+      permission:
+        columns:
+          - created_at
+          - created_by_user_id
+          - id
+          - is_deleted
+          - project_id
+          - project_type_id
+          - updated_at
+          - updated_by_user_id
+        filter: {}
+  update_permissions:
+    - role: moped-admin
+      permission:
+        columns:
+          - is_deleted
+          - project_id
+          - project_type_id
+        filter: {}
+        check: null
+        set:
+          updated_by_user_id: x-hasura-user-db-id
+    - role: moped-editor
+      permission:
+        columns:
+          - is_deleted
+          - project_id
+          - project_type_id
+        filter: {}
+        check: null
+        set:
+          updated_by_user_id: x-hasura-user-db-id
+  event_triggers:
```

Comment:
do we still need to keep these triggers around?

Reply:
thank you for catching this! yes, there is some cleanup on permissions that I'll roll in here as well. No roles need update or insert permissions either.